### PR TITLE
Feature/blog cards section

### PR DESCRIPTION
**Pull request by BigRedSquare**

### CODE REVIEW

File: src/components/Contact.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Contact = ({
             iconImageAdditionalClass={['!h-6 !w-6']}
         />
         <div className="relative mt-8">
-            <Image url={contactFormImage} additionalClasses={['w-full']} />
+            <Image url={contactFormImage} additionalClasses={['w-full !rounded-3xl']} />
```
Comment (CiotkaCierpienia) [inline src/components/Contact.js:23]: use `rounded="3xl"` instead of importants in classes

File: src/components/ProjectCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                        dangerouslySetInnerHTML={{ __html: excerpt }}
+                        className="line-clamp-5 lg:line-clamp-3 lg:line-clamp-4 font-lora text-sm
+                        xl:text-base uppercase italic mb-10 lg:mb-5 xl:mb-10"
+                    />
+                </Card.Text>
+                <div className="relative mr-5 w-fit">
+                    <Button
+                        label="Explore"
+                        size="lg"
+                        additionalClasses={['bg-secondary font-archivo font-normal '
```
Comment (CiotkaCierpienia) [inline src/components/ProjectCard.js:34]: use variant="secondary" instead of bg-secondary

File: src/sections/ProjectCards.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { navigate } from 'gatsby';
+import ProjectCard from '../components/ProjectCard';
+
+const ProjectCards = ({ projects }) => (
+    <div className="flex flex-col">
+        {projects.map((project) => (
+            <ProjectCard
+                key={project.id}
+                onClick={() => { navigate(`/${project.slug}`); }}
```
Comment (CiotkaCierpienia) [inline src/sections/ProjectCards.js:10]: use <Link to={`/${project.slug}`}> instead of onClick

Comment (BigRedSquare) [reply src/sections/ProjectCards.js]: Link used

File: src/components/ProjectCard.js (hunk 2 above)
Comment (BigRedSquare) [reply src/components/ProjectCard.js]: variant used

**Pull request merged**